app/views/record/view.html.erb

```diff
@@ -1,63 +1,9 @@
 <%= content_for(:title, "Search Results | MIT Libraries") %>
 
-<%= field_string(@record, 'error') %>
+<%= render(partial: 'shared/error', collection: @errors) %>
 
-<h1><%= @record['title'] %></h1>
-
-<%= field_object(@record, 'alternate_titles' ) %>
-
-<%= field_list(@record, 'call_numbers' ) %>
-
-<%= field_string(@record, 'citation' ) %>
-
-<%= field_list(@record, 'content_type' ) %>
-
-<%= field_list(@record, 'contents' ) %>
-
-<%= field_table(@record, 'contributors', %w[value kind affiliation identifier mit_affiliated]) %>
-
-<%= field_table(@record, 'dates', %w[kind value range note]) %>
-
-<%= field_string(@record, 'edition') %>
-
-<%= field_list(@record, 'file_formats') %>
-
-<%= field_string(@record, 'format') %>
-
-<%= field_table(@record, 'funding_information', %w[award_number award_uri funder_identifier funder_identifier_type funder_name]) %>
-
-<%= field_table(@record, 'holdings', %w[call_number collection format location note]) %>
-
-<%= field_string(@record, 'id') %>
-
-<%= field_object(@record, 'identifiers') %>
-
-<%= field_list(@record, 'languages') %>
-
-<%= field_table(@record, 'links', %w[kind text url restrictions]) %>
-
-<%= field_string(@record, 'literary_form') %>
-
-<%= field_table(@record, 'locations', %w[kind value geopoint]) %>
-
-<%= field_object(@record, 'notes') %>
-
-<%= field_string(@record, 'numbering') %>
-
-<%= field_string(@record, 'physical_description') %>
-
-<%= field_list(@record, 'publication_frequency') %>
-
-<%= field_list(@record, 'publication_information') %>
-
-<%= field_table(@record, 'related_items', %w[description item_type relationship uri]) %>
-
-<%= field_table(@record, 'rights', %w[description kind uri]) %>
-
-<%= field_string(@record, 'source') %>
-
-<%= field_string(@record, 'source_link') %>
-
-<%= field_object(@record, 'subjects') %>
-
-<%= field_list(@record, 'summary') %>
+<% if @record.nil? %>
+	<%= render('record_empty') %>
```

Comment:
This seems like odd whitespace. I think the linters run away from the `erb` files so I'm not positive on this.

Reply:
Oh, yeah - this is tabbed and not spaced. Fixing...